release-notes/opensearch.release-notes-2.7.0.md

```diff
@@ -21,6 +21,7 @@
 - Add retry delay as dynamic setting for cluster maanger throttling. ([#6998](https://github.com/opensearch-project/OpenSearch/pull/6998))
 - Introduce full support for searchable snapshots ([#5087](https://github.com/opensearch-project/OpenSearch/issues/5087))
 - Introduce full support for Segment Replication ([#5147](https://github.com/opensearch-project/OpenSearch/issues/5147))
+- Introduce a new field type: flat_object to help prevent mapping explosions ([#6507](https://github.com/opensearch-project/OpenSearch/pull/6507))
```

Comment:
How about linking to the [original issue](https://github.com/opensearch-project/OpenSearch/issues/1018) here instead of the PR? That's probably a better resource to learn about the feature than the PR.

Reply:
smart. done.